utilities/src/test_data.rs

```diff
@@ -371,35 +331,35 @@ pub mod bbs_coherent_context_test_data {
             "https://schema.org/",
             "https://w3id.org/vc-revocation-list-2020/v1"
         ],
-        "id": "6d6bfa18-8cfe-40ec-9775-a0878e6f305a",
+        "id": "743875ce-8ec0-4b6a-a67d-33a64392a5d3",
```

Comment:
```suggestion
        "id": "uuid:743875ce-8ec0-4b6a-a67d-33a64392a5d3",
```

does this work here?

Reply:
This should also work, but at the moment we're creating the proof requests with uuid's without a prefix, so we should update test data and id generation.